hw4/ParseTree/Operand.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace ParseTree;
+
+using System;
+
+/// <summary>
+/// Класс элемента дерева
+/// </summary>
+public class Operand : INode
+{
+    /// <summary>
+    /// Значение элемента 
+    /// </summary>
+    public int operand { get; set; }
```

Comment:
```suggestion
    public int Operand { get; set; }
```
Свойства всегда-всегда именуются с заглавной. И я бы назвал его как-нибудь типа Value, а то Operand.Operand выглядит довольно странно